### PR TITLE
Ireland - Richard Yun

### PR DESCRIPTION
**Pull request by richardyun**
ireland

### CODE REVIEW

**Comment by dpaschal-lf** [comment]
please make successive screen shots focused on the area being reviewed

File: flags.css [hunk 1]
```diff
@@ -3,6 +3,25 @@
 /* Colombia */
 
 /* Ireland */
+#ireland {
+    display: flex;
+    height: 200px;
```
Comment (dpaschal-lf) [inline flags.css]: change the flag container so you don't have to keep doing this for every id.

File: flags.css [hunk 2]
```diff
@@ -3,6 +3,25 @@
...
+    width: 300px;
+    border: 1px solid black
+}
+#ireland > .leftSegment, #ireland > .middleSegment, #ireland > .rightSegment {
+    display: inline-block;
```
Comment (dpaschal-lf) [inline flags.css]: there's no point in making it inline-block.  flex children will change size

File: flags.html [hunk 3]
```diff
@@ -11,7 +11,9 @@
 
         </div>
         <div id="ireland" class="flagContainer">
-
+            <div class="leftSegment green"></div>
```
Comment (dpaschal-lf) [inline flags.html]: again... why with the color classes that you aren't using?

**Review by dpaschal-lf — CHANGES_REQUESTED** [review]
similar problems to before, and also a needless display property

**Comment by richardyun** [comment]
ireland

File: flags.css [hunk 4]
```diff
@@ -1,8 +1,30 @@
...
+}
 /* Indonesia */
 
 /* Colombia */
 
 /* Ireland */
+#ireland {
+    display: flex
+}
+#ireland>.leftSegment, #ireland>.middleSegment, #ireland>.rightSegment {
```
Comment (dpaschal-lf) [inline flags.css:16]: could have also said 
```
#ireland > *
```

**Review by dpaschal-lf — APPROVED** [review]

**Pull request merged**